integration-test/src/main/java/com/sequenceiq/it/IntegrationTestApp.java

```diff
@@ -52,7 +52,7 @@
 
 @EnableAutoConfiguration(exclude = {DataSourceAutoConfiguration.class, DataSourceTransactionManagerAutoConfiguration.class,
         HibernateJpaAutoConfiguration.class})
-@ComponentScan(basePackages = "com.sequenceiq.it")
+@ComponentScan(basePackages = { "com.sequenceiq.it", "com.sequenceiq.cloudbreak.cloud.gcp.util" })
```

Comment:
Why do we need this?

Reply:
because now we have a bean and not a static class